analyzers/tests/SonarAnalyzer.UnitTest/TestCases/AssertionArgsShouldBePassedInCorrectOrder.MsTest.cs

```diff
@@ -32,3 +32,30 @@ public void Foo()
         }
     }
 }
+
+// https://github.com/SonarSource/sonar-dotnet/issues/6547
+namespace Repro_6547
+{
+    [TestClass]
+    class Program
+    {
+        public enum Seasons
+        {
+            Spring,
+            Summer,
+            Autumn,
+            Winter
+        }
+
+        [TestMethod]
+        public void Repro_6547(Seasons seasonEnum, string seasonString)
```

Comment:
Please make the test work. This is a parameterized test and requires different attributes to actually work.
https://learn.microsoft.com/en-us/dotnet/core/testing/unit-testing-with-mstest#add-more-features

https://learn.microsoft.com/en-us/visualstudio/test/using-microsoft-visualstudio-testtools-unittesting-members-in-unit-tests?view=vs-2022#attributes-used-for-data-driven-testing